resources/x509_certificate.rb

```diff
@@ -0,0 +1,134 @@
+provides :openssl_x509 # legacy_name
+provides :openssl_x509_cert
```

Comment:
Do we need this resource name as well?

Reply:
openssl_x509 war for backward compatibility. I guess we should keep it. 

I was afraid openssl_x509_certificate was maybe too long so I added openssl_x509_cert... Maybe that's not  a good idea. I can remove it if needed.